notebooks/Sigma_test_Notebook.ipynb

```diff
@@ -0,0 +1,358 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "single_rule.es_query"
+   ]
+  },
+  {
+   "source": [
+    "# Text Sigma examples\n",
+    "\n",
+    "The API also provides the option to provide a Sigma rule by text and the backend will parse it with the installed Sigma mappings. This can be especially helpful when developing new rules before installing and exposing them to all the other user of a Timesketch instance."
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "MOCK_SIGMA_RULE = \"\"\"\n",
+    "title: Suspicious Installation of Zenmap\n",
+    "id: 5266a592-b793-11ea-b3de-0242ac130004\n",
+    "description: Detects suspicious installation of Zenmap\n",
+    "references:\n",
+    "    - https://rmusser.net/docs/ATT&CK-Stuff/ATT&CK/Discovery.html\n",
+    "author: Alexander Jaeger\n",
+    "date: 2020/06/26\n",
+    "modified: 2021/01/01\n",
+    "logsource:\n",
+    "    product: linux\n",
+    "    service: shell\n",
+    "detection:\n",
+    "    keywords:\n",
+    "        # Generic suspicious commands\n",
+    "        - '*apt-get install zmap*'\n",
+    "    condition: keywords\n",
+    "falsepositives:\n",
+    "    - Unknown\n",
+    "level: high\n",
+    "\"\"\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rule_by_text = ts_client.get_sigma_rule_by_text(MOCK_SIGMA_RULE)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rule_by_text.es_query"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rule_by_text.references"
+   ]
+  },
+  {
+   "source": [
+    "# Analyzer\n",
+    "\n",
+    "Timesketch also has a Sigma analyzer that will add labels to all matching events. To do so you need a Timeline object to run the analyzer. The Analyzer then will take all rules installed on the Timesketch instance and go over the Timeline."
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "timelines = sketch.list_timelines()\n",
+    "timeline = None\n",
+    "for timeline_ in timelines:\n",
+    "  if timeline_.name == 'sigma_events.csv':\n",
+    "    timeline = timeline_\n",
+    "    break"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = timeline.run_analyzer(analyzer_name='sigma', ignore_previous=True)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sketch.get_analyzer_status()\n",
+    "\"\"\"This should show something like:\n",
```

Comment:
take this comment out of the code cell and put it in a markdown cell instead....